lib/test.js

```diff
@@ -21,7 +21,7 @@ var Test = function(options) {
   }
   
   this.web3 = new Web3();
-  this.web3.setProvider(this.sim.provider());
+  this.web3.setProvider(this.sim.provider(options));
```

Comment:
The testrpc provider is [configurable]( https://github.com/ethereumjs/testrpc/blob/master/README.md#library), and I found it useful to change things like the debugging and logging levels to help figure out why some tests were failing.

This could probably be moved to a key on the options object (e.g. `options.providerOptions || {}`)